### ISSUE TITLE
sorry to bring this up again but not changing colors only dimming

### ISSUE THREAD

**Issue by adarkenigma** [mpolednik/script.kodi.hue.ambilight#77]
I am using SPMC 16.7 on shield original with 2 Phillips hue lightstrip plus (2nd gen) and bridge. I installed Kodi Phillips hue 0.8b2 add-on and requests 2.12.4 add-on autoconfig found bridge fills in IP address and user info. I got light ID's 3 and 4 (using old hue iOS app)
Under "main" tab mode is set to ambilight, under "ambilight" tab I have two light. 
Minimum brightness 20 and 70 max. Use old algorithm checked. Thresholds are at 0.25 and 0.33 (don't know what its for so didn't mess with it I hope someone can elaborate on it). Color Bias is at 18. 

Under setting >Video>Acceleration>MediaCodec (surface) is unchecked.

now on any playback its just dims lights when video starts and undims once playback stops.

what am I doing wrong?

I also added to advanced settings <usedroidprojectioncapture>true</usedroidprojectioncapture> found on koying's wiki. (shield asked me SPMC wants to capture screen I selected never show again and allowed it) I am also using recommended advanced settings from koying's wiki on 4k vizio tv.

restarted kodi clear cache, restarted shield few times just to be safe.

can someone write definite guide for all android boxes.

**Comment by MagicMan84** [comment]
Hi, did you ever get this sorted? 

I have the exact same issue. I’m sure I’ve changed every setting combination at least ten times. Nothing seems to work.

**Comment by adarkenigma** [comment]
dev of add-on has moved on to kodi 17 so I thought it best to move on until koying release final v17, also Phillips will also open up its API later early 2018 or so I read.

**Comment by MagicMan84** [comment]
Thanks for replying.

**Comment by MagicMan84** [comment]
Hi, don't know if your interested. But I just managed to get it working using the instructions on this page: https://github.com/koying/script.kodi.hue.ambilight

**Comment by adarkenigma** [comment]
yes I was able to make it work in past but ended up uninstalling it.
 module.requests add-on is pre-request and so is <usedroidprojectioncapture>true</usedroidprojectioncapture> under video section in advancedsettings.xml but if you have 4k UHD TV like I do it requires you to disable SPMC HQ scalers and pictures quality looked bad so I figured I'll just wait for final SPMC v17 release and try again.
not sure if ambilight users are in koying's priority list.
 
I am happy you were able to make it work.
